packages/sources/coinmetrics/src/adapter.ts

```diff
@@ -78,7 +79,7 @@ export const makeWSHandler = (config?: Config): MakeWSHandler => {
           url,
         }
       },
-      shouldRetryConnection: (closeContext) => {
+      shouldRetryConnection: (closeContext: WebsocketErrorMessageSchema) => {
```

Comment:
Won't even need the type definition here because `shouldRetryConnection` is already typed. Yay TypeScirpt.